### DISCUSSION TITLE
봇 서버 수 삭제 요청

### DISCUSSION THREAD

**Discussion by JellyBrick** [despenser08/KRList-Stats#84] [Delete Request] [answered]
`서버 수가 급락하는 부분`을 지워주실 수 있나요?

ID: `479108435984515072`

**Comment by despenser08** [comment, marked as answer]
처리가 완료되었습니다.
![image](https://user-images.githubusercontent.com/50207925/135723000-3238af20-83ef-408d-9779-2b91464ad0c9.png)
